src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -1934,8 +1934,15 @@ static void _copyAttr(SvgNode* to, const SvgNode* from)
 }
 
 
-static void _cloneNode(SvgNode* from, SvgNode* parent)
+static void _cloneNode(SvgNode* from, SvgNode* parent, int depth)
 {
+    /* Exception handling: Prevent invalid SVG data input.
+       The size is the arbitrary value, we needs a experimental size of it. */
+    if (depth == 8192) {
+        TVGERR("SVG", "Infinite recursive call = %d! A current SVG data may be wrongly written!\n", depth);
```

Comment:
my suggestion, but as u wish:
Infinite recursive call - stopped after %d calls! Svg file may be incorrectly formatted.

Reply:
@mgrudzinska  applied with your suggestion. thanks.